### PR TITLE
Kubernetes health check helpers

### PR DESCRIPTION
**Pull request by TymoteuszGach**

### CODE REVIEW

File: healthstatusprovider.go [hunk 1]
```diff
@@ -0,0 +1,166 @@
+package probe
```
Comment (dmitry-onishchenko) [inline healthstatusprovider.go]: Shall we follow some repo layout, e.g. https://github.com/golang-standards/project-layout ?

Comment (TymoteuszGach) [reply healthstatusprovider.go]: good idea 👍

**Review by dmitry-onishchenko — APPROVED** [review]

**Comment by dmitry-onishchenko** [comment]
LGTM for me

File: README.md [hunk 2]
```diff
@@ -1 +1,9 @@
...
\ No newline at end of file
+# Vamp Health Probe
+
+This repository contains helper structs to run Kubernetes health checks in parallel.
+
+## Running unit tests
+
+```bash
+make test
+```
```
Comment (rkolappin) [inline README.md:46]: Only minor comment: perhaps it'd be good to show an example how to use / implement the helper

**Review by rkolappin — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (TymoteuszGach) [reply README.md:46]: Good point, I added an example.

**Pull request merged**